src/webots/scene_tree/WbSceneTree.cpp

```diff
@@ -1119,8 +1119,9 @@ void WbSceneTree::updateSelection() {
       // baseNode = NULL if none or multiple instances exists
       baseNode = baseNode->getSingleFinalizedProtoInstance();
 
-    if (baseNode && !baseNode->areWrenObjectsInitialized())
+    if (baseNode && !(baseNode->areWrenObjectsInitialized() || baseNode->isPostFinalizedCalled()))
```

Comment:
Not sure about the implication, but wouldn't it be possible to keep only the ``isPostFinalizedCalled`` conditionand remove the ``areWrenObjectsInitialized``.

Reply:
Yes, I think there won't be any difference. I will check.